topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -871,10 +877,224 @@ These datasets can be used for the downstream analysis, e.g. mapping.
+> > Over time, however, the shorter fragments are becoming rarer and thus more long fragments attach to pores and are sequenced.
+> {: .solution }
+{: .question}
+
+The read length over experiment time should be stable.
+It can slightly increase over the time as short fragments tend to be over-sequenced at the beginning and are less present over the time.
+
+![Read length over experiment time](../../images/quality-control/read_length_over_experiment_time-pycoqc.png "Read length over experiment time")
+
+## Channel activity over time
+
+It gives an overview of available pores, pore usage during the experiment, inactive pores and shows if the loading of the flow cell is good (almost all pores are used).
+In this case, the vast majority of channels/pores are inactive (white) throughout the sequencing run, so the run can be considered as bad.
+
+You would hope for a plot that it is dark near the X-axis, and with higher Y-values (increasing time) doesn’t get too light/white.
+Depending if you chose “Reads” or “Bases” on the left the colour indicates either number of bases or reads per time interval
+
+![Channel activity over time](../../images/quality-control/channel_activity_over_time-pycoqc.png "Channel activity over time")
+
+> ### {% icon details %} Other "Channel activity over time" profile
+>
+> In this example, almost all pores are active all along the run (yellow/red profile) which indicate an excellent run.
+>
+> ![Channel activity over time good profile](../../images/quality-control/channel_activity_over_time-pycoqc-good.png)
+>
+{: .details}
+
+
+> ### {% icon comment %} Try it on!
+> Do the quality control with **FastQC** {% icon tool %} and/or **Nanoplot** {% icon tool %} on `nanopore_basecalled-guppy.fastq.gz` and compare the results!
+{: .comment}
+
 # Conclusion
 {:.no_toc}
 
-In this tutorial we checked the quality of FASTQ files to ensure that their data looks good before inferring any further information. This step is the usual first step for analyses such as RNA-Seq, ChIP-Seq, or any other OMIC analysis relying on NGS data. Quality control steps are similar for any type of sequencing data:
+In this tutorial we checked the quality of FASTQ files to ensure that their data looks good before inferring any further information.
+This step is the usual first step for analyses such as RNA-Seq, ChIP-Seq, or any other OMIC analysis relying on NGS data.
+Quality control steps are similar for any type of sequencing data:
 
-- Quality assessment with tools like **FASTQE** {% icon tool %} and **FastQC** {% icon tool %}
-- Trimming and filtering with a tool like **Cutadapt** {% icon tool %}
+- Quality assessment with tools like **FASTQE** {% icon tool %}, **FastQC** {% icon tool %}, **Nanoplot** {% icon tool %} and **PycoQC** {% icon tool %}
```

Comment:
maybe it would be good to split up by which tool is appropriate for which situation just to re-enforce that a bit? Maybe something like this?

```suggestion
- Quality assessment with tools like:
  - *Short Reads*: {% tool [FASTQE](toolshed.g2.bx.psu.edu/repos/iuc/fastqe/fastqe/0.2.6+galaxy2) %}
  - *Long Reads*:  {% tool [Nanoplot](toolshed.g2.bx.psu.edu/repos/iuc/nanoplot/nanoplot/1.28.2+galaxy1) %}
  - *Short+Long*: {% tool [FASTQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.73+galaxy0) %}
  - *Nanopore only*: {% tool [PycoQC](toolshed.g2.bx.psu.edu/repos/iuc/pycoqc/pycoqc/2.5.2+galaxy0) %}
```